<launch>

<node name="blue_serial" pkg="qbo_communicator" type="blue_serial" output="screen"/>

<node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>
    <param name="wavepath" value="$(find qbo_communicator)/sounds"/>

  <node name="qbo_communicator" pkg="qbo_communicator" type="blue_com.py" output="screen"/>
    
    <remap from="cmd_vel" to="/cmd_vel"/>
    <param name="scale_linear" value="0.5" type="double"/>
    <param name="scale_angular" value="1.5" type="double"/>
    <param name="max_speed" value="0.3"/>
    <param name="start_speed" value="0.1"/>
    <param name="linear_increment" value="0.05"/>
    <param name="angular_increment" value="0.4"/>
    
  

</launch>
